Foody/Home/Category.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Category.swift
+//  Foody
+//
+//  Created by Ezgi  on 30.01.2023.
+//
+
+import Foundation
+import UIKit
+
+struct Category{
+    let title:String
+    let image:UIImage
+}
+let categories:[Category] = [
```

Comment:
Su sekilde tanimlarsak daha guzel gozukur.
```suggestion
let categories: [Category] = [
Category(title: "Beef", image: UIImage(named: "chicken1")),
Category(title: "Breakfast", image: UIImage(named: "chicken1")),
Category(title: "Chicken", image:UIImage(named: "chicken1") ),
Category(title: "Dessert", image: UIImage(named: "chicken1")),
Category(title: "Goat", image: UIImage(named: "chicken1")),
Category(title: "Lamb", image: UIImage(named: "chicken1")),
Category(title: "Miscellaneous", image: UIImage(named: "chicken1")),
Category(title: "Pasta", image: UIImage(named: "chicken1")),
Category(title: "Pork", image: UIImage(named: "chicken1")),
Category(title: "Seafood", image: UIImage(named: "chicken1")),
Category(title: "Side", image: UIImage(named: "chicken1")),
Category(title: "Starter", image: UIImage(named: "chicken1")),
Category(title: "Vegan", image: UIImage(named: "chicken1")),
Category(title: "Vegatarian", image:UIImage(named: "chicken1"))
]
```